packages/edit-site/src/components/post-list/index.js

```diff
@@ -402,6 +400,14 @@ export default function PostList( { postType } ) {
 				onChangeView={ setView }
 				selection={ selection }
 				onChangeSelection={ onChangeSelection }
+				isClickable={ ( item ) => item.status !== 'trash' }
+				handleClick={ ( { id } ) => {
```

Comment:
I believe this should be `onClick` right?

Reply:
Thanks for catching this! Fixed with c9ab19d095f3211c082a075472702df575938fe1